e2e/tests/integrations/integrator/remove-organizer-uitpas-integration.test.ts

```diff
@@ -0,0 +1,25 @@
+import { test, expect, Page } from "@playwright/test";
+import { IntegrationType } from "@app-types/IntegrationType";
+import { createIntegrationAsIntegrator } from "./create-integration.js";
+import { requestActivationAsIntegrator } from "./request-activation.js";
+import { addUiTPASOrganizer } from "./add-organizer-uitpas-integration.test.js";
+
+test("As an integrator I can remove an organizer from an UiTPAS integration", async ({
+  page,
+}) => {
+  const { integrationId } = await createIntegrationAsIntegrator(
+    page,
+    IntegrationType.UiTPAS
+  );
+  await requestActivationAsIntegrator(
+    page,
+    integrationId,
+    IntegrationType.UiTPAS
+  );
+  await addUiTPASOrganizer(page);
```

Comment:
Strange that you are importing a helper from another test. It would be better to add a separate file.